frontend/public/components/edit-yaml.jsx

```diff
@@ -385,7 +435,7 @@ const EditYAMLInner = (props) => {
     if (retryObjs) {
       const yamlDocuments = retryObjs.map((obj) => convertObjToYAMLString(obj));
       setDisplayResults(false);
-      getEditor().setValue(yamlDocuments.join('---\n'));
+      getEditor()?.setValue(yamlDocuments.join('---\n'));
```

Comment:
Why are these checks needed now and were not needed earlier? Is this because of redirection and setting of code automatically? Can we not delay the rendering of the component itself until the editor is initialized correctly?

Reply:
The /import route is used for 2 different pages. One page has the Import YAML Editor and the other page shows the list of resources created from the editor. Now when the user is on this list page, if he tries to import code from OLS, the OLS sees that he is already in the /import route, therefore does not redirect. But it is unable to find the editor, therefore the application crashes. 
IMO, the routes should have been different for the 2 pages, which of course is a different bug/discussion. 
In this feature work, adding those "?" tends to prevent the app from crashing for now.